WooCommerce/Classes/Notifications/PushNotificationsManager.swift

```diff
@@ -51,6 +51,9 @@ final class PushNotificationsManager: PushNotesManager {
         }
     }
 
+    private var siteID: Int64? {
+        ServiceLocator.stores.sessionManager.defaultStoreID
```

Comment:
⚠️  There's an injected `storesManager` via `configuration`:

```suggestion
        configuration.storesManager.sessionManager.defaultStoreID
```

Reply:
thanks for the suggestion! this made it possible to DI the `storesManager`. updated in https://github.com/woocommerce/woocommerce-ios/pull/2320/commits/7620bb0da4f901ebfd141939e5511a9ab657c39e